en/performance/feature-tuning.html

```diff
@@ -183,7 +187,7 @@ <h2 id="tuning-query-performance-for-lexical-search">Tuning query performance fo
       <strong>Avoid using large posting lists all together</strong>:
       Setting <a href="../reference/schema-reference.html#weakand-stopword-limit">stopword-limit</a> to 0.6,
       ensures that all terms that are estimated to occur in more than 60% of the documents are considered stopwords
-      and dropped entirely from the query, also from ranking.
+      and dropped entirely from the query and also from ranking.
```

Comment:
@geirst : Some might ask what happens with stopwords in phrase search here, if parts of the phrase is dropped. but this is for a weakAnd, so that does not apply. we can consider clarifying this somewhere in the text, someone will ask about this sometime ...